app/views/badges/redeem.slim

```diff
@@ -1,9 +1,9 @@
 .container
   .row
     .col-md-12
-      .form-container
-        h2
-          | Obter crachá
+      h3
+        | OBTER BADGE_
```

Comment:
No, I'm experimenting using underscores on the headers, just to see how it fits with the rest of the UI :)